extensions/opentelemetry/opentelemetry/deployment/src/test/java/io/quarkus/opentelemetry/deployment/VertxOpenTelemetryTest.java

```diff
@@ -38,19 +46,29 @@
 public class VertxOpenTelemetryTest {
     @RegisterExtension
     static final QuarkusUnitTest unitTest = new QuarkusUnitTest()
+            .overrideConfigKey("quarkus.opentelemetry.propagators", "trace-context,baggage")
+            .overrideConfigKey("quarkus.opentelemetry.tracer.resource-attributes", "service.name=test")
```

Comment:
to properly test this we would need an `application.properties` in the test directory as I can't recall how the method is used and how it differs from reading config from a properties file directly

Reply:
I don't think we need since the service name value now is being fetch from the `ApplicationInfoBuildItem` and there the default is `<<unset>>`, so with this override we are actually testing that setting the resource attribute as a name has its value being used and is overriding the one that is by the default of application name. And this overriding behavior is the same one being used in the OTEL autoconfigure sdk

Reply:
Right, but I was thinking it would be good to handle testing the `resource-attribute` aspect of key/value pairs from an `application.properties` file instead of an override, to make sure the parsing of multiple values works as expected.